src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -1408,7 +1408,7 @@ public async Task RudeEdits_SourceGenerators()
 class C { int Y => 1; }
 ";
             var sourceV2 = @"
-/* GENERATE: class G { int X2 => 1; } */
+/* GENERATE: class G { double X1 => 1; } */
```

Comment:
>double [](http://example.com/codeflow?start=23&length=6)

Maybe use "add generic type parameter" rude edit here as well, in anticipation of changing member signatures being allowed in near future.

Reply:
Good idea